index.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const NR = require('node-resque');
+
+// SET UP THE CONNECTION
+const connectionDetails = {
+    pkg: 'ioredis',
+    host: process.env.REDIS_HOST,
+    password: process.env.REDIS_PASSWORD,
+    port: process.env.REDIS_PORT,
+    database: 0
+};
+
+// DEFINE YOUR WORKER TASKS
+const jobs = {
+    start: {
+        plugins: ['jobLock', 'retry'],
+        pluginOptions: {
+            jobLock: {},
+            retry: {
+                retryLimit: 3,
+                retryDelay: (1000 * 5)
+            }
+        },
+        /* config looks like this
+            executor:
+                name: k8s-vm
+                options:
+                    kubernetes: ...
+            buildConfig:
+                apiUri:
+                buildId:
+                container:
+                token:
+         */
+        perform: (config, callback) => {
+            // eslint-disable-next-line
+            const ExecutorPlugin = require(`screwdriver-executor-${config.executor.name}`);
```

Comment:
I missed a few days, so I may have missed something in the middle. Last we talked, we had decided on dedicated workers. This line suggests that the "queue executor" is going to still use the router pattern. Did we decide to keep the router pattern for the first pass?